cmd/pinniped-proxy/Cargo.toml

```diff
@@ -11,4 +11,8 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
Oh - one other dependency that I started using here. https://docs.rs/anyhow/1.0.34/anyhow/ makes it even easier to handle different types of errors with context etc.. I'll explain a bit more in a later PR.